samples/dynamodb/lambda/DynamoDB_1/Sales/Sales.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <AWSProjectType>Lambda</AWSProjectType>
+    <!-- This property makes the build directory similar to a publish directory and helps the AWS .NET Lambda Mock Test Tool find project dependencies. -->
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <!-- Generate ready to run images during publishing to improve cold start time. -->
+    <PublishReadyToRun>true</PublishReadyToRun>
+    <LangVersion>8.0</LangVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Amazon.Lambda.Core" Version="2.1.0" />
+    <PackageReference Include="Amazon.Lambda.Serialization.SystemTextJson" Version="2.3.1" />
+    <PackageReference Include="NServiceBus.AwsLambda.SQS" Version="1.0.0" />
+    <PackageReference Include="NServiceBus.Persistence.DynamoDB" Version="1.0.0" />
```

Comment:
```suggestion
    <PackageReference Include="Amazon.Lambda.Core" Version="2.*" />
    <PackageReference Include="Amazon.Lambda.Serialization.SystemTextJson" Version="2.*" />
    <PackageReference Include="NServiceBus.AwsLambda.SQS" Version="1.*" />
    <PackageReference Include="NServiceBus.Persistence.DynamoDB" Version="1.*" />
```

Reply:
I realize we have the SystemTextJson pinned to a minor. Maybe we can align that as well https://github.com/Particular/docs.particular.net/blob/master/samples/hosting/aws-lambda-sqs/SQSLambda_1/AwsLambda.SQSTrigger/AwsLambda.SQSTrigger.csproj#L10